### PR TITLE
Disable SQ 5.2 precise issues report api

### PR DESCRIPTION
**Pull request by mpaladin**

### CODE REVIEW

File: java-squid/src/main/java/org/sonar/java/model/VisitorsBridge.java [hunk 1]
```diff
@@ -284,7 +284,7 @@ public void addIssue(File file, JavaCheck check, int line, String message) {
      * FIXME(mpaladin) DO NOT GO ON RELEASE WITH THIS CONSTANT SET TO TRUE
      *
      * **/
-    private static final boolean ENABLE_NEW_APIS = true;
+    private static final boolean ENABLE_NEW_APIS = false;
```
Comment (benzonico) [inline java-squid/src/main/java/org/sonar/java/model/VisitorsBridge.java]: Fixme comment should also be removed

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Private method 'textSpanFor' is never used.](https://github.com/SonarSource/sonar-java/blob/76f11c528ec03277bc739fd120b99cd10804a921/java-squid/src/main/java/org/sonar/java/model/VisitorsBridge.java#L315) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Private method 'textSpanFor' is never used.](https://github.com/SonarSource/sonar-java/blob/76f11c528ec03277bc739fd120b99cd10804a921/java-squid/src/main/java/org/sonar/java/model/VisitorsBridge.java#L315) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Private method 'textSpanFor' is never used.](https://github.com/SonarSource/sonar-java/blob/341468768ac6c978c08dec46fcd9131146c17a3b/java-squid/src/main/java/org/sonar/java/model/VisitorsBridge.java#L315) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

**Pull request merged**